pycbc/psd/analytical_space.py

```diff
@@ -711,3 +1902,99 @@ def analytical_psd_lisa_tdi_AE_confusion(length, delta_f, low_freq_cutoff,
     fseries = psd_AE + psd_X_confusion
```

Comment:
Could you show me where I can find this approximation (or assumption)?

Reply:
@HumphreyWang By definition,
![image](https://github.com/gwastro/pycbc/assets/6574526/33882f81-01ae-4ccf-a220-6ef1fe3eba32)
Those confusion noise PSDs are for X channel, so just add them, the CSD of X, Y channels `S_xy` is well below the PSD of X at low-frequency region of confusion noise, so we can ignore the effect of confusion noise on `S_xy`. For T channel, it contributes very little to SNR, so I also ignore modification of it.

Reply:
This assumption has a good match with LDC data:
![image](https://github.com/gwastro/pycbc/assets/6574526/080833a5-7c44-4e9f-be16-7969386b5571)

Reply:
This assumption has a good match with LDC data:
![image](https://github.com/gwastro/pycbc/assets/6574526/afb1bcb2-f25f-4ed3-a64a-3348b35f682e)